packages/web/src/ui/views/parameterControls.js

```diff
@@ -1,127 +1,167 @@
 const html = require('bel')
```

Comment:
I like the refactoring and changes. However, this file is just way too big.

I was wondering if this could be broken into the functional pieces as well. This will allow test cases to be created against each of the functional pieces as well.

Reply:
Leaving as-is for now, I'll open an issue to refactor this.